core/committer/txvalidator/v20/validator.go

```diff
@@ -406,6 +406,7 @@ func (v *TxValidator) validateTx(req *blockValidationRequest, results chan<- *bl
 				return
 			}
 
+			logger.Debugw("Config transaction passed validation checks", "channel", channel)
```

Comment:
This could be a bit misleading as for the config transaction, both the actual config validation and apply happens inside `Apply` function below. May be you want to enhance the message a bit - "Config transaction envelop passed validation checks"

Reply:
+1 and by extension the next log entry should read "transaction validated and applied.."

Reply:
Thanks guys, done!